gluten-ut/spark32/src/test/resources/sql-tests/results/udf/udf-window.sql.out

```diff
@@ -0,0 +1,382 @@
+-- Automatically generated by SQLQueryTestSuite
```

Comment:
Could you explain a bit why we need this file?

Reply:
Maybe we can convert the string in the result file to Array[Row] according to the schema for comparison. This part has many modifications and can be placed in another independent PR.

Reply:
I see. Which number in this file is changed, could you clarify? I did not find `1.1539890888012805` and `0.7057890433107311` in this file.

Reply:
There are two different rows, the penultimate column.
```
< 2	a	2	1	1	3	4	1.3333333333333333	0.5773502691896258	NULL	1	NULL	2	2	2	4	3	1.0	1.0	2	4	0.22222222222222224	0.33333333333333337	2	4.772185885555555E8	1.0	0.5773502691896258	0.4714045207910317	[1,1,2]	[1,2]	1.1539890888012811	-0.6672217220327235
---
> 2	a	2	1	1	3	4	1.3333333333333333	0.5773502691896258	NULL	1	NULL	2	2	2	4	3	1.0	1.0	2	4	0.22222222222222224	0.33333333333333337	2	4.772185885555555E8	1.0	0.5773502691896258	0.4714045207910317	[1,1,2]	[1,2]	1.1539890888012805	-0.6672217220327235
299c299
< 3	b	3	1	1	3	6	2.0	1.0	1	1	1	3	3	3	3	3	1.0	1.0	2	3	0.6666666666666666	1.0	35.3687091175E8	1.0	1.0	0.816496580927726	[1,2,3]	[1,2,3]	0.7057890433107324	-1.4999999999999984
---
> 3	b	3	1	1	3	6	2.0	1.0	1	1	1	3	3	3	3	3	1.0	1.0	2	3	0.6666666666666666	1.0	35.3687091175E8	1.0	1.0	0.816496580927726	[1,2,3]	[1,2,3]	0.7057890433107311	-1.4999999999999984
```